recipes/r-sdcmicro/meta.yaml

```
@@ -0,0 +1,126 @@
+{% set version = '5.6.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-sdcmicro
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/sdcMicro_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/sdcMicro/sdcMicro_{{ version }}.tar.gz
+  sha256: a1447a5a5d5a3af04925afc5e2b664db51aa97459c13dfdb7542e915c9e0aa8a
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-dt
+    - r-mass
+    - r-rcpp
+    - r-vim >=4.7.0
+    - r-car
+    - r-cardata
+    - r-cluster
+    - r-data.table
+    - r-e1071
+    - r-ggplot2
+    - r-haven
+    - r-knitr
+    - r-prettydoc
+    - r-rhandsontable
+    - r-rmarkdown
+    - r-robustbase
+    - r-shiny >=1.4.0
+    - r-shinybs
+    - r-xtable
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-dt
+    - r-mass
+    - r-rcpp
+    - r-vim >=4.7.0
+    - r-car
+    - r-cardata
+    - r-cluster
+    - r-data.table
+    - r-e1071
+    - r-ggplot2
+    - r-haven
+    - r-knitr
+    - r-prettydoc
+    - r-rhandsontable
+    - r-rmarkdown
+    - r-robustbase
+    - r-shiny >=1.4.0
+    - r-shinybs
+    - r-xtable
+
+test:
+  commands:
+    - $R -e "library('sdcMicro')"           # [not win]
+    - "\"%R%\" -e \"library('sdcMicro')\""  # [win]
+
+about:
+  home: https://github.com/sdcTools/sdcMicro
+  license: GPL-2.0-or-later

```

cran meta data `# License: GPL-2` evaluates to
```suggestion
  license: GPL-2.0-only

```